src/common/envalid.ts

```diff
@@ -1,6 +1,22 @@
-import { config } from 'dotenv'
+import { DotenvParseOutput, parse } from 'dotenv'
 import { bool, cleanEnv, json, str } from 'envalid'
-import { existsSync } from 'fs'
+import { existsSync, readFileSync } from 'fs'
+
+export const dotEnvParse = (path: string): DotenvParseOutput => {
```

Comment:
I think that this function should not update process.env 
The single responsibility is to parse env file and return parsed content

Reply:
Isn't it desirable to add it to `process.env`?